renderdoc/android/jdwp.cpp

```diff
@@ -213,8 +213,29 @@ bool InjectLibraries(const std::string &deviceID, Network::Socket *sock)
 
     if(vulkanLoaderMethod)
     {
-      int32_t slotIdx =
-          conn.GetLocalVariable(vulkanLoaderClass, vulkanLoaderMethod, "librarySearchPath");
+      std::vector<VariableSlot> slots =
+          conn.GetLocalVariables(vulkanLoaderClass, vulkanLoaderMethod);
+
+      int32_t slotIdx = -1, thisSlotIdx = -1;
+      bool slot4Exists = false;
+
+      for(const VariableSlot &s : slots)
+      {
+        if(s.name == "librarySearchPath")
+        {
+          slotIdx = s.slot;
+          break;
+        }
+        else if(s.name == "this")
+          thisSlotIdx = s.slot;
```

Comment:
Style nit-pick: I prefer having braces around all if/else branches, so in this case because the if needs a brace scope then the else should get one too.